tests/basic/volume_ops_test.go

```diff
@@ -2843,6 +2843,111 @@ var _ = Describe("{NFSProxyVolumeValidation}", func() {
 	})
 })
 
+var _ = Describe("{SharedVolFuseTest}", func() {
+	/*
+
+				https://portworx.atlassian.net/browse/PWX-35639
+			   https://portworx.atlassian.net/browse/PTX-21805
+
+
+		       	1. Deploy apps  with mix of sv4 service and sv4  using k8s
+		  		2.Get the list of coordinator nodes have sv4 volume attached
+				3.Stop/Start PX on the coordinator node
+				4.Check the failover has happened i.e. we  have new  coordinator node assigned after step 3
+		   		5.Repeat this in a loop for 10 iterations
+
+
+
+	*/
+	var contexts []*scheduler.Context
+	JustBeforeEach(func() {
+		StartTorpedoTest("SharedVolFuseTest", "Validate PX operations after sharedv4 and sharedv4 svc volumes  failover multiple times", nil, 0)
+	})
+
+	It("schedule sharedv4 and sharedv4_svc volumes and perform failover of the coordinator node", func() {
+
+		stepLog = "create sharedv4 and sharedv4_svc apps "
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			appList := Inst().AppList
+
+			defer func() {
+				Inst().AppList = appList
+			}()
+
+			Inst().AppList = []string{"vdbench-sv4-svc", "fio-sharedv4"}
+			contexts = make([]*scheduler.Context, 0)
+
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("pxfusetest-%d", i))...)
+			}
+
+			for _, ctx := range contexts {
+				log.InfoD("Validating application [%s]", ctx.App.Key)
+				ValidateContext(ctx)
+			}
+		})
+
+		stepLog = "restart PX on coordinator nodes"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			for _, ctx := range contexts {
+				vols, err := Inst().S.GetVolumes(ctx)
+				log.FailOnError(err, "failed to get volumes from the contexts")
+
+				for _, vol := range vols {
+					log.InfoD("Initiating failover iteration for volume [%s] of app [%s]", vol.Name, ctx.App.Key)
+					for i := 1; i <= 5; i++ {
+						log.Infof("Iteration: #%d for volume [%s]", i, vol.Name)
+						attachedNode, err := Inst().V.GetNodeForVolume(vol, 1*time.Minute, 5*time.Second)
+						log.FailOnError(err, "error getting attached node for volume [%s]", vol.Name)
+						log.Infof("vol [%s] is attached to node [%s]", vol.Name, attachedNode.Name)
```

Comment:
How do you make sure, active IO happening on the device. Options would be 
root@ip-10-13-232-218:~# iostat -xm 5 | grep pxd585034700205937734
pxd/pxd585034700205937734    0.00      0.00     0.00  20.01    2.00     5.47    0.74      0.00     0.35  32.03    6.15     6.40    0.00      0.00     0.00   0.00    0.00     0.00    0.00   0.09


pxd/pxd585034700205937734    0.00      0.00     0.00   0.00    0.00     0.00    5.40      0.03     2.60  32.50    0.63     5.63    0.00      0.00     0.00   0.00    0.00     0.00    0.00   0.56
pxd/pxd585034700205937734    0.00      0.00     0.00   0.00    0.00     0.00    1.60      0.01     1.60  50.00    0.38     8.00    0.00      0.00     0.00   0.00    0.00     0.00    0.00   0.16

OR 
root@ip-10-13-232-218:~# cat /sys/block/pxd\!pxd585034700205937734/inflight 
       0        0

If there is any other way in PX-Test  you can use that as well.

Reply:
@Santhoshairody 
Currently we are running vdbench and fio apps which does continuous IOs. Also since we are restarting all nodes now it should cover the data path. I think adding this check will introduce unnecessary complexity to the test and make test slower. Please let me know you thoughts